javascript/new-src/src/js/.jshintrc

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
How fussy is the config file, can you put all the options in one file and just ignore some of them?